snap/gui/musescore.desktop

```diff
@@ -0,0 +1,17 @@
+[Desktop Entry]
+Version=1.1
+Name=MuseScore
+GenericName=Music notation
+GenericName[de]=Notensatz
+GenericName[fr]=Notation musicale
+Comment=Create, play and print sheet music
+Comment[ru]=Визуальный редактор нотных партитур
+Comment[fr]=Gravure de partitions musicales
+Exec=musescore.mscore %F
+Icon=${SNAP}/usr/share/icons/hicolor/scalable/apps/mscore.svg
+StartupNotify=true
+Terminal=false
+Type=Application
+Categories=Qt;Audio;Sequencer;Midi;AudioVideoEditing;Music;AudioVideo;
+MimeType=application/x-musescore;application/x-musescore+xml;application/vnd.recordare.musicxml;application/vnd.recordare.musicxml+xml;
```

Comment:
this here is much shorter than that from build/Linux+BSD/mscore.desktop.in, why?